commons.opam

```diff
@@ -28,6 +28,8 @@ depends: [
   "ppx_hash" {>= "v0.14.0"}
   "parmap" {>= "1.2.4"}
   "odoc" {with-doc}
+  "digestif" {>= "1.0.0"}
```

Comment:
Sorry about the inconvenience, but can you merge the latest develop?
Those things are now specified in the dune-project file.